src/taskManagement/models/taskManagementManger.ts

```diff
@@ -8,17 +8,24 @@ import { IFindInactiveTasksRequest, IGetTaskResponse, IRetrieveAndStartRequest }
 
 @injectable()
 export class TaskManagementManager {
-  private repository?: TaskRepository;
+  private repository: TaskRepository;
 
-  public constructor(@inject(Services.LOGGER) private readonly logger: ILogger, private readonly connectionManager: ConnectionManager) {}
+  public constructor(@inject(Services.LOGGER) private readonly logger: ILogger, private readonly connectionManager: ConnectionManager) {
```

Comment:
the is no guarantee that the repository will be initialized before it is used this way